index.html

```diff
@@ -0,0 +1,129 @@
+        </lord-icon>
+        <h2>Web Development</h2>
+        <p>Great design is more than just aesthetics. It can change the way we feel about a company and the products and
+          services they offer. </p>
+      </div>
+      <div class="w-card">
+        <lord-icon src="https://cdn.lordicon.com/vnioezyi.json" trigger="loop-on-hover" delay="100"
+          colors="outline:#ffffff,primary:#ec5242" class="work-icons">
+        </lord-icon>
+        <h2>Web Hosting</h2>
+        <p>Our goal is to provide outstanding services for the best possible price. </p>
+      </div>
+      <div class="w-card">
+        <lord-icon src="https://cdn.lordicon.com/nrxxmbbt.json" trigger="loop-on-hover" delay="100"
+          colors="primary:#606874,secondary:#ec5242,tertiary:#ebe6ef" class="work-icons">
+        </lord-icon>
+        <h2>Domain Registration</h2>
+        <p>Online success starts with a great domain. At Afghan Graphics, we make it easy and inexpensive to get the
+          domain you want, fast!</p>
+      </div>
+      <div class="w-card">
+        <lord-icon src="https://cdn.lordicon.com/hiqmdfkt.json" trigger="hover"
+          colors="primary:#ec5242,secondary:#cb5eee" class="work-icons">
+        </lord-icon>
+        <h2>Digital Branding</h2>
+        <p>To stand out, brands must have personality. They must express themselves in a unique way to be easily
+          recognizable. We work with you to define your brand's.</p>
+      </div>
+      <div class="w-card">
+        <lord-icon src="https://cdn.lordicon.com/luglsinl.json" trigger="loop-on-hover" delay="100"
+          colors="primary:#ffffff,secondary:#ec5242" class="work-icons">
+        </lord-icon>
+        <h2>Network Management</h2>
+        <p>We review the components of your existing network and make sound recommendations for improvements to the network's design and performance.</p>
+      </div>
+    </div>
+    <button class="join-button">Contact Us</button>
+  </main>
+
+  <section class="our-team">
+    <h1 class="heading text-black">Our Team</h1>
+    <hr class="hr-line">
+    <div class="team">
+      
+    </div>
+  </section>
+
+  <footer>
+    <div class="partners">
+      <h1 class="heading">Partners</h1>
+      <hr class="hr-line">
+      <ul class="d-flex justify-content-center align-items-center flex-wrap gap-4">
+        <li><img src="./assets/images/oracle-logo.png" width="100px" alt=""></li>
+        <li><img src="./assets/images/Google_2015_logo.svg" width="100px" alt=""></li>
+        <li><img src="./assets/images/Mozilla_Foundation_logo_(pre_2017).svg.png" width="100px" alt=""></li>
+        <li><img src="./assets/images/jb_beam.png" width="100px" alt=""></li>
+        <li><img src="./assets/images/Lenovo_logo_2015.svg.png" width="100px" alt=""></li>
+      </ul>
+    </div>
+    <div class="text-center my-2">
+      <h3><i class="fa-solid">&#xf1f9;</i> Copyright
+      <br> Some Rights Reserved
+  </h3>
+  </div>
+  </footer>
```

Comment:
- [ ] Kindly make sure your footer design for both **Home Page** && **About Page** sticks as much as possible to the template design. Your footer is supposed to also include the **logo** so for example once clicked it can immediately lead you to the hero section. Just take look at the **[template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)** and make yours similar to it.

### Template Design
![Screenshot from 2022-11-24 00-02-42](https://user-images.githubusercontent.com/83042742/203653716-ac5de81d-2c47-4547-b433-ba224d5896c2.png)

### Your Design
![Screenshot from 2022-11-24 00-02-59](https://user-images.githubusercontent.com/83042742/203653743-787570d9-14bf-4a28-9ef7-ee94cd9299b0.png)